### PR TITLE
update file source

### PR DESCRIPTION
**Pull request by wcy-fdu**
## Info
This pr:
- add parquet encode for s3 and gcs source.
- align the missing fields on both sides.

And  @tabVersion could you please add `Source file name as column` part or assign to the implementer of this feature(Not sure if it's you, don't mind)

And I notice that their is another related [PR](https://github.com/risingwavelabs/risingwave-docs/pull/2453) for structural changes, feel free to merge it and I will apply the changes.
## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`emile-00`, `hengm3467`, & `WanYixian`).

### CODE REVIEW

**Comment by wcy-fdu** [comment]
Note that `PARQUET` encode is **not** included in 1.10, so we need to put it in 1.11(dev).

**Comment by tabVersion** [comment]
> Note that `PARQUET` encode is **not** included in 1.10, so we need to put it in 1.11(dev).

I guess you mean 2.0

File: docs/ingest/ingest-from-s3.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ slug: /ingest-from-s3
   <link rel="canonical" href="https://docs.risingwave.com/docs/current/ingest-from-s3/" />
 </head>
 
-Use the SQL statement below to connect RisingWave to an Amazon S3 source. RisingWave supports both CSV and [ndjson](https://ndjson.org/) file formats.
+Use the SQL statement below to connect RisingWave to an Amazon S3 source. RisingWave supports both CSV and ndjson file formats.
```
Comment (ShanlanLi) [inline docs/ingest/ingest-from-s3.md]: Delete this link because the website is wrong now:
image

**Review by ShanlanLi — APPROVED** [review]
LGTM, thanks very much for helping resolve this pr!

**Comment by ShanlanLi** [comment]
> > Note that `PARQUET` encode is **not** included in 1.10, so we need to put it in 1.11(dev).
> 
> I guess you mean 2.0

Yes, the 1.11 in our doc now is 2.0. We plan to change 1.11 to 2.0 when versioning docs later, since the tag in the doc issue is 1.11

File: docs/ingest/ingest-from-s3.md (hunk 1 above)
Comment (WanYixian) [inline docs/ingest/ingest-from-s3.md]:
```suggestion
Use the SQL statement below to connect RisingWave to an Amazon S3 source. RisingWave supports both CSV and [ndjson](https://github.com/ndjson) file formats.
```

**Review by WanYixian — APPROVED** [review]
It's true that "http://ndjson.org/" expires, shall we replace it with its Github home page?

File: docs/ingest/ingest-from-s3.md (hunk 1 above)
Comment (tabVersion) [reply docs/ingest/ingest-from-s3.md]: SGTM

Comment (ShanlanLi) [reply docs/ingest/ingest-from-s3.md]: I add them now.

**Comment by ShanlanLi** [comment]
@tabVersion  I wonder if this pr is ready to be merged? Do we still need to add the `Source file name as column part` congyi mentioned in the info?

> And @tabVersion could you please add Source file name as column part or assign to the implementer of this feature(Not sure if it's you, don't mind)

**Comment by tabVersion** [comment]
> @tabVersion  I wonder if this pr is ready to be merged? Do we still need to add the `Source file name as column part` congyi mentioned in the info?
> 
> 
> 
> > And @tabVersion could you please add Source file name as column part or assign to the implementer of this feature(Not sure if it's you, don't mind)

You can merge this one. I will submit a new PR later.

**Pull request merged**